src/core/datatypes/phylogenetics/character/DnaState.cpp

```diff
@@ -408,6 +408,10 @@ RbBitSet DnaState::getState(void) const
     switch ( state )
     {
         case '-':
+            bs.set(0);
```

Comment:
This change should probably be reverted now.

Reply:
Why should this be reverted? The current code treats the missing states the same as '?', and 'N'. Should we treat these differently?